README.md

```diff
@@ -38,6 +38,13 @@ $ cd netplugin; make demo
 $ vagrant ssh netplugin-node1
 ```
 
+Optionally, variables can be passed to Makefile if needed. For example, to
+use 4 GB memory for the vagrant VMs, run:
+
+```
+CONTIV_MEMORY=4096 make demo
+```
+
```

Comment:
did you forget CONTIV_CPUS ?  would be nice to show the defaults as well in the docs

Reply:
not a big deal, but CONTIV_MEMORY_MB would be clearer on the unit

Reply:
If we change this, we should also change all the other Vagrantfiles to keep them consistent

Reply:
> did you forget CONTIV_CPUS ? would be nice to show the defaults as well in the docs

@chrisplo Added CONTIV_CPUS too in the latest commit.  Also added the default values for `CONTIV_CPUS` and `CONTIV_MEMORY` in `README.md`.

> not a big deal, but CONTIV_MEMORY_MB would be clearer on the unit

@chrisplo - As @dseevr said, `CONTIV_MEMORY ` matches and is consistent with the variable name for Docker swarm in https://github.com/contiv/netplugin/blob/master/Vagrantfile#L33. `CONTIV_MEMORY` for both Docker swarm and k8s can be renamed to `CONTIV_MEMORY_MB` in the future if needed.

Reply:
Good idea, we can hit all of them at once in another PR if desired